MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -372,7 +374,11 @@ public Task<MediaInfo> GetMediaInfo(MediaInfoRequest request, CancellationToken
 
             string analyzeDuration = string.Empty;
 
-            if (request.MediaSource.AnalyzeDurationMs > 0)
+            if (!string.IsNullOrEmpty(_ffmpegAnalyzeDuration))
+            {
+                analyzeDuration = "-analyzeduration " + _ffmpegAnalyzeDuration;
+            }
+	    else if (request.MediaSource.AnalyzeDurationMs > 0)
```

Comment:
```suggestion
            else if (request.MediaSource.AnalyzeDurationMs > 0)
```

(I think I got the spacing correct)